editor/components/block-list/style.scss

```diff
@@ -280,36 +326,40 @@
 		}
 	}
 
-	// Left and right side UI
-	> .editor-block-settings-menu,
-	> .editor-block-mover {
+
+	/**
+	 * 	Left and right side UI
+	 */
+
+	.editor-block-settings-menu,
+	.editor-block-mover {
 		position: absolute;
 		top: 0;
 		padding: 0;
 	}
 
 	// Right side UI
-	> .editor-block-settings-menu {
-		right: $block-mover-margin;
+	.editor-block-settings-menu {
+		right: -$block-mover-margin -$block-padding + 2px;
```

Comment:
Operators get a bit confusing here; one's a negation, the other's a subtraction, but they're applied the same (where it could be confused as a shorthand syntax e.g. `margin: -5px -10px`).

At its most verbose, I could see this expressed as: `#{ -1 * $block-mover-margin } - $block-padding + 2px`

(Not really a suggestion, but it's how I'd express it)

Reply:
This rule worked: 

`right: #{ -1 * $block-mover-margin - $block-padding + 2px };`

Does that address your concern?

Reply:
>Does that address your concern?

Sure 👍 Assumes one remembers their order of operations though. Tend to paranthesize on mixed operators.